docs/04-reference/wingsdk-spec.md

```diff
@@ -454,9 +454,9 @@ resource Queue {
   inflight push(message: Json): void;
 
   /**
-   * Pops (deletes and returns) a message from the queue.
+   * Pops (deletes and returns) a message from the queue if any.
    */
-  inflight pop(): Json;
+  inflight pop(): Json?;
 
```

Comment:
there is some discrepancy between the spec and the IQueueClient, where spec pushes and pops `Json` but IQueueClient goes with `string`. LMK which one we should fix

Reply:
I think long term we want to use `Json` since it's closer to a future where we support queues of user-defined structs. But keeping the current implementation with strings is okay for now, since switching to Json will require a breaking change